client/src/transmitter/cast/websocket.ts

```diff
@@ -0,0 +1,158 @@
+import isIp from 'is-ip'
+
+const isLocalHost = (hostname) => {
+  return hostname === 'localhost' || hostname.endsWith('.localhost')
+}
+
+const { hostname, port, protocol } = location
+
+const localhost = isLocalHost(hostname) || isIp(hostname)
```

Comment:
Rename it to just `local` or more precisely `localhostOrIp`?

Reply:
`islocalOrIp`